module/os/linux/zfs/zvol_os.c

```diff
@@ -41,17 +41,80 @@
 #include <linux/blkdev_compat.h>
 #include <linux/task_io_accounting_ops.h>
 
-static unsigned int zvol_major = ZVOL_MAJOR;
-static unsigned int zvol_request_sync = 0;
-static unsigned int zvol_prefetch_bytes = (128 * 1024);
-static unsigned long zvol_max_discard_blocks = 16384;
-static unsigned int zvol_threads = 32;
-static const unsigned int zvol_open_timeout_ms = 1000;
+#ifdef HAVE_BLK_MQ
+#include <linux/blk-mq.h>
+#endif
+
+static void zvol_request_impl(zvol_state_t *zv, struct bio *bio,
+    struct request *rq, boolean_t force_sync);
+
+unsigned int zvol_major = ZVOL_MAJOR;
+unsigned int zvol_request_sync = 0;
+unsigned int zvol_prefetch_bytes = (128 * 1024);
+unsigned long zvol_max_discard_blocks = 16384;
+unsigned int zvol_open_timeout_ms = 1000;
```

Comment:
why did you destaticify and deconstify these? the visibility/usage scope and constness didn't change

Reply:
I think I messed it up fixing a merge conflict, will fix